src/module/show-popup-comment.js

```diff
@@ -29,13 +29,94 @@ async function showPopupComment(id) {
   p1.textContent = `${data.meals[0].strMeal}`;
   meal.appendChild(p1);
 
-  const p2 = document.createElement('p');
-  p2.textContent = `Category: ${data.meals[0].strCategory}`;
-  meal.appendChild(p2);
+  const IngredientsDiv = document.createElement('div');
+  IngredientsDiv.className = 'Ingredients';
+  meal.appendChild(IngredientsDiv);
 
-  const p3 = document.createElement('p');
-  p3.textContent = `Area: ${data.meals[0].strArea}`;
-  meal.appendChild(p3);
+  const Ingredients = document.createElement('ul');
+  IngredientsDiv.appendChild(Ingredients);
+
+  const IngredientsItems1 = document.createElement('li');
+  Ingredients.appendChild(IngredientsItems1);
+
+  const imgIngredients1 = document.createElement('img');
```

Comment:
That's look very cool when you show ingredients in the comment-popup window! If we will have time, let's think how to make this code more smart: not to write in code Ingredient1, Ingredient2, ... IngredientN. But read it in "loop", like `strIngedient`+i. By the way, there could be up to 20 ingredients.

Reply:
Thank you for the feedback, I did that because in the array of objects the Ingredients are separated, and they're not on one array, I tried to implement a loop but its does work as supposed I will try to find another way, thank you again. 👍